pkg/transformer/openshift/openshift.go

```diff
@@ -135,6 +135,24 @@ func getAbsBuildContext(context string, composeFileDir string) (string, error) {
 		return "", err
 	}
 	prefix := strings.Trim(string(out), "\n")
+	index := strings.Index(context, prefix)
+	if index != -1 {
+		if prefix == "" {
```

Comment:
`prefix == ""` is depend on `index != -1` but both has separate return statement too, so I didn't put them in  `index != -1 && prefix == ""` condition